src/generate-route-tiles.js

```diff
@@ -0,0 +1,63 @@
+const { readFile } = require('node:fs/promises');
+const { join } = require("path");
+const bbox = require('@turf/bbox').default;
+const geojson2mvt = require('geojson2mvt');
+
+// Converts the `routes` property on a stop point to a comma-separated string
+// Mutates the input geojson in-place
+// We do this because the final VectorTile protobufs do not support arrays for feature properties
+function stopRoutesToCSVInPlace(geojson) {
+  for(const feature of geojson.features){
+    if(feature.geometry.type === 'Point'){
+      const routes = feature.properties.routes;
+      feature.properties['route_ids'] = routes.map((route) => route.route_id).join(',');
+      delete feature.properties.routes;
+    } 
+  }
+}
+
+async function generateRouteTiles(
+  zippedGtfsPath,
+  outputPath,
+) {
+  const gtfsToGeoJSON = await import('gtfs-to-geojson');
```

Comment:
Why do you import asynchronously? I see no advantage. This is server-side code; we already have the library installed.